<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_cosine_similarity</class>
 <widget class="QWidget" name="widget_cosine_similarity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>2</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Search most relevant tweets</string>
  </property>
  <property name="statusTip">
   <string extracomment="search"/>
  </property>
  <property name="whatsThis">
   <string extracomment="search"/>
  </property>
  <widget class="QWidget" name="widget_querry_container" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">

QWidget#widget_querry_container { 
	
	background-color: rgb(255, 255, 255);
     border: 1px solid gray; 
     border-radius: 7px; 
 } </string>
   </property>
   <widget class="QLineEdit" name="line_edit_querry">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_fetch_tweets">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="text_edit_most_relevant_tweets">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>521</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
